helix-gateway/src/main/java/org/apache/helix/gateway/util/StateTransitionMessageTranslator.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.helix.gateway.util;
+
+public class StateTransitionMessageTranslator {
+
+  public static void translateSTMsgToProto() {
```

Comment:
I would assume this should be interface instead of concrete class.

interface I am expecting was:


fromHelixToProto
fromProtoToHelix

Reply:
I am thinking it should a util class with abstract functions?

Reply:
Per offline sync, Updated the naming.